.github/workflows/contracts.yml

```diff
@@ -18,6 +18,7 @@ concurrency:
 
 env:
   RUST_LOG: info,libp2p=off
+  GH_TOKEN: ${{ github.token }}
```

Comment:
Since we're using wget now can we remove this?

Reply:
agree, removed in [3cc95e0](https://github.com/EspressoSystems/espresso-sequencer/pull/1329/commits/3cc95e028d73534fa20beee5bb0970e62b6a5e91)